Makefile

```diff
@@ -1,12 +1,35 @@
-PORT?=8008
-LASTTAG = $(shell git describe --tags --abbrev=0)
+PORT     ?= 8008
+VENV     := venv/bin
+VENV_DEV := venv-dev/bin
+VENV_MIN := venv-min/bin
+CONF     := conf/glances.conf
+PIP      := $(VENV)/pip
+PYTHON   := $(VENV)/python
+UNITTEST := unittest
+
+DOCKER_BUILD      := docker buildx build
+DOCKER_RUN        := docker run
+DOCKERFILE_UBUNTU := docker-files/ubuntu.Dockerfile
+DOCKERFILE_ALPINE := docker-files/alpine.Dockerfile
+PODMAN_SOCK       := /run/user/1000/podman/podman.sock
```

Comment:
Instead of hard coding 1000, could you please use the output of `id -u` for PODMAN_SOCK.

Reply:
Thanks for the feedback; it's done.